### DISCUSSION TITLE
Windows Theme compatibility

### DISCUSSION THREAD

**Discussion by Scarlaid** [seerge/g-helper#1976] [Q&A] [answered]
If you use `ui_mode: light` you can theme G-Helper as long as you, can, I guess.
I found out that the selection dropdown doesn't blend into the theme, anyway to make them work?

![image](https://github.com/seerge/g-helper/assets/90851437/b6349182-7494-4df3-b544-e868811f9a17)

**Comment by seerge** [comment]
@Scarlaid is it some 3rd party software overriding default settings ? If you want to use dark ui you need to set ``ui_mode : "dark", `` in config (since you started editing that in a first place). Then it would use white icons

**Reply by Scarlaid** [reply]
This is custom theming, injected from SecureUXTheme 
I intentionally changed to light theme bcoz of the reason, but see some elements being white (not mentioning the fan speed graph), so I thought this is somewhat about the dropdown being not consistent with the UI

**Reply by Scarlaid** [reply]
So yeah, I just asked if this dropdown is intentional design or just me

**Reply by seerge** [reply, marked as answer]
@Scarlaid in light theme everything is intentionally light :) Rest - is up to your 3rd-party app and what it does.

**Reply by Scarlaid** [reply]
Unfortunate. Thanks for the clarification, I thought it was just design's fault
